backends/shellout.go

```diff
@@ -0,0 +1,54 @@
+package backends
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"os/signal"
+)
+
+func shellout(execBinary string, args ...string) ([]byte, error) {
+	cmd := exec.Command(execBinary, args...)
+	output, err := cmd.Output()
+	if err != nil {
+		return nil, err
+	}
+	return output, nil
+}
+
+func delegate(execBinary string) {
+	cmd := exec.Command(execBinary, os.Args[1:]...)
+	cmd.Stdin = os.Stdin
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	cmd.Env = os.Environ()
```

Comment:
nit: I wonder if this line is useful 🤔

Reply:
indeed, if nil uses the parent process env :)